Gamedata/Bluedog_DB/Parts/AtlasV/bluedog_AtlasV_Star5F.cfg

```diff
@@ -44,6 +44,7 @@ PART
 	bulkheadProfiles = srf
 
 	tags = sepratron perfectron separation booster push solid kick star 5 atlas ldc
+	techtag = atlasV
```

Comment:
Needs to go into an LDC node

Reply:
Although technically a modern motor, predecessors such as the Star 5C date back to 60s so its not that anachronistic either to move it earlier.